lib/routes/index.js

```diff
@@ -12,7 +11,8 @@ const merge = require('deepmerge')
 /**
  * @typedef {import('fastify').RouteShorthandOptions} RouteShorthandOptions
  * @typedef {import('fastify').FastifySchema} FastifySchema
- *
+ * @typedef {import('../providers/cartobio').OrganismeCertificateur} OrganismeCertificateur
```

Comment:
Idéalement importer les types des fichiers typescripts, je me souviens avoir eu des problèmes de boucles d'import.

Reply:
Ok noté :)